src/subtitling/Group setups/github.md

```diff
@@ -0,0 +1,145 @@
+# Using GitHub for fansubbing
+
+When working in a group,
+you'll often find multiple people making changes to the same or multiple files.
+The Translator will need a place to upload their translation to,
+the typesetter(s) may need a place where they can upload their typesetting to,
+etc.
+For this kind of operation,
+a Git environment may be preferable.
+With Git you can very easily do version control,
+which allows you to see who made changes to what files and when,
+and also create an easy place to find all the relevant files and their contents
+without requiring users to blindly download files to edit them
+the same way they would when working in an FTP setup.
+
+The benefit to using a Git platform like GitHub over FTPs can be extremely beneficial,
+but may also require users to be more tech-savvy individually than with an FTP setup,
+and for there to be someone to the side who can solve problems that may arise.
+Additionally,
+having an FTP available may still be a good idea for your encoders,
+since they will still need a place to upload bigger files to,
+which GitHub is not suited for.
+
+## Setting up a GitHub repository
+
+To begin,
+you'll want everyone involved to create an account on either [GitHub](GitHub_home)
+or an alternative Git platform like [GitLab](GitLab_home) or [BitBucket](BitBucket_home),
+depending on where your project leader decides to host the repositories.
+For this guide,
+we will be assuming you're using GitHub.
+
+If you are the project leader,
+you will likely want to be the one to create a repository.
+For bigger groups,
+we recommend you start an organization.
+This will allow you to keep better track of your ongoing projects,
+as well as assign more administrators who can create and maintain repositories.
+You can create a repository by pressing the + button on the top-right of the page and clicking "New repository".
+
+![Create a new repository](../images/gh_make_repo1.png)
```

Comment:
To have captions on images, you must do it [as here](https://github.com/Irrational-Encoding-Wizardry/guide.encode.moe/blob/12d54c908f1f262833209ba07ad57df59305d338/src/encoding/video-artifacts.md#L50) by adding an italicized span on the next line.

Reply:
I think it's generally self-explanatory what they cover, I just added those captions because my linter complained :teehee: